jest.config.js

```diff
@@ -0,0 +1,8 @@
+//jest.config.js
```

Comment:
Lint errors:

  1:1   error  Expected space or tab after '//' in comment           spaced-comment
  4:3   error  Strings must use singlequote                          quotes
  5:1   error  This line has a length of 125. Maximum allowed is 80  max-len
  5:5   error  Strings must use singlequote                          quotes
  5:92  error  Strings must use singlequote                          quotes
  6:5   error  Strings must use singlequote                          quotes
  6:28  error  Strings must use singlequote                          quotes
  6:48  error  Missing trailing comma                                comma-dangle
  7:4   error  Missing trailing comma                                comma-dangle
  8:2   error  Missing semicolon                                     semi

Please fix or exclude this file from linting.

Reply:
Added a rule to exclude the max-len lint error. Fixed other error